bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -327,5 +327,5 @@ spec:
   maturity: alpha
   provider:
     name: IBM
-  replaces: ibm-common-service-operator.v3.7.1
+  replaces: ibm-common-service-operator.v3.7.2
```

Comment:
Should be changed to 3.7.3 once it gets released

Reply:
Sure, update to `3.7.3`, and `3.8.0` has been move to `v3` channel, but let hold it until `3.7.3` is released.